instance/schemas/theming.py

```diff
@@ -19,74 +19,122 @@
 """
 Theming relates schemas for validating theme configurations.
 """
-from django.core.exceptions import ValidationError
-from schema import Schema, Regex, Optional, Or, SchemaError
+from jsonschema import validate
 
-color = Regex(r'^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$')
+color_regex = '^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$'
 
 
-def nullable(schema):
```

Comment:
Can we keep allowing the frontend to set a null value?
The registration form doesn't have any theming fields and doesn't send the key when the user is registering.

Reply:
@giovannicimolin, this is now implemented.